### PR TITLE
18 parent profile

### PR DESCRIPTION
**Pull request by AlaaTaima**
Relates #18

### CODE REVIEW

File: client/src/components/common/Card/index.js [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
 
-export default Card;
+  componentDidMount() {
+    // we will make a request to fetch data by using (Axios)
+  }
+
+  render() {
+    const { data } = this.state;
+    return (
+      <>
```
Comment (11janci) [inline client/src/components/common/Card/index.js]: use `<React.Fragment>`

File: client/src/components/common/StudentTable/index.js [hunk 2]
```diff
@@ -1,36 +1,54 @@
...
+          <table className="parentTable">
+            <thead className="parentTable__title">
+              <tr>
+                <th className="parentTable__title1">Name</th>
+                <th>Grade</th>
+                <th className="parentTable__title2">Class</th>
+              </tr>
+            </thead>
+            <tbody className="parentTable__content">
+              {data.map(({ name, grade, StudentClass, id }) => {
```
Comment (11janci) [inline client/src/components/common/StudentTable/index.js]: use camelCase for local variables and object properties (`StudentClass` -> `studentClass`)

File: client/src/components/pages/Activities/index.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Card from '../../common/Card';
+import './index.css';
+
+const Activities = () => {
+  return (
+    <div className="ActivityName">
+      <p className="ActivityName__tiltle">Class Activities</p>
```
Comment (11janci) [inline client/src/components/pages/Activities/index.js]: typo in title; also pls use either camelCase or kebab-case for class names (pick one as a team and stay consistent): `ActivityName` -> `activityName`

File: client/src/components/common/Card/index.css [hunk 4]
```diff
@@ -13,16 +13,19 @@
...
   padding: 0.5rem;
   align-items: center;
-  border-top-left-radius: 1rem;
-  border-bottom-left-radius: 1rem;
+  justify-content: center;
+  border-radius: 1rem;
 }
 
-.card-box__title > .title {
+.card-box__title >.title {
```
Comment (11janci) [inline client/src/components/common/Card/index.css]: the `>` selector is not needed in this case. I would actually got rid of the `.title` class altogether (it's quite generic name and there could be conflicts) and use `.card-box__title h5` instead.

File: client/src/components/common/Card/index.js (hunk 1 above)
Comment (AlaaTaima) [reply client/src/components/common/Card/index.js]: what's the difference ??

File: client/src/components/pages/Activities/index.js (hunk 3 above)
Comment (AlaaTaima) [reply client/src/components/pages/Activities/index.js]: ok , We will do that.

File: client/src/components/common/Card/index.js (hunk 1 above)
Comment (11janci) [reply client/src/components/common/Card/index.js]: TBH I wasn't familiar with the empty tag and I thought it was a mistake :) With that said, it does look like a mistake and also can be easily overlooked, so I would still go with the fragment, but that's just my personal preference, so up to you..

**Pull request merged**